platform_versioned_docs/version-24.2/enterprise/configuration/authentication.mdx

```diff
@@ -129,7 +129,7 @@ This must be an `https://` URI, per Microsoft's requirements.
     ```bash
     TOWER_OIDC_CLIENT=<YOUR_APPLICATION_ID>
     TOWER_OIDC_SECRET=<YOUR_CLIENT_CREDENTIALS_SECRET>
-    TOWER_OIDC_ISSUER=<YOUR_OIDC_METADATA_URL_UP_TO_"v2.0">   (e.g. https://login.microsoftonline.com/000000-0000-0000-00-0000000000000/v2.0)
+    TOWER_OIDC_ISSUER=<YOUR_OIDC_METADATA_URL_UP_TO_"v2.0">  (e.g. https://login.microsoftonline.com/000000-0000-0000-00-0000000000000/v2.0)
```

Comment:
```suggestion
    TOWER_OIDC_ISSUER=<OIDC_METADATA_URL_UP_TO_"v2.0">  (e.g. https://login.microsoftonline.com/000000-0000-0000-00-0000000000000/v2.0)
```
I would drop `YOUR` from all of these. However, there are other instances of `YOUR` on this page. If we change one we should change all on this page, at least on the page or section we are working on.